internal/backends/gl/event_loop.rs

```diff
@@ -289,6 +289,7 @@ pub enum CustomEvent {
     WakeEventLoopWorkaround,
     UpdateWindowProperties(winit::window::WindowId),
     UserEvent(Box<dyn FnOnce() + Send>),
+    WindowHidden,
```

Comment:
Please add a comment

```suggestion
    /// Called from the `GLWindow::hide` so that we can check if we should quit the event loop
    WindowHidden,
```